GVFS/GVFS.Common/NuGetUpgrade/NuGetFeed.cs

```diff
@@ -176,7 +177,7 @@ private void SetSourceRepository()
             this.sourceRepository = Repository.Factory.GetCoreV3(this.feedUrl);
             if (!string.IsNullOrEmpty(this.personalAccessToken))
             {
-                this.sourceRepository.PackageSource.Credentials = BuildCredentialsFromPAT(this.personalAccessToken);
+                this.sourceRepository.PackageSource.Credentials = BuildCredentialsFromPAT(this.personalAccessToken, !this.platformSupportsEncryption);

```

Comment:
What's the advantage of having the `platformSupportsEncryption` member variable vs. using `GVFSPlatform.Instance.UnderConstruction.SupportsNuGetEncryption` here?

Reply:
It could have gone there as well, but I wanted to keep the GVFSPlatform out of NuGetFeed (and not introduce static references in that component)